wezterm-gui/src/overlay/prompt.rs

```diff
@@ -66,7 +66,7 @@ pub fn show_line_prompt_overlay(
     term.render(&[Change::Text(text)])?;
 
     let mut host = PromptHost::new();
-    let mut editor = LineEditor::new(&mut term);
+    let mut editor = LineEditor::with_text(&mut term, &args.with_content);
```

Comment:
I think we can achieve this this way instead:

```suggestion
    let mut editor = LineEditor::new(&mut term);
    if let Some(value) = &args.initial_value {
      editor.set_line_and_cursor(value, value.len());
   }
```